examples/use-cases/update_devices_by_csv/README.md

```diff
@@ -0,0 +1,52 @@
+# Update Automox Devices by CSV
```

Comment:
Maybe this description could be written a little clearer? This updates the group of a device, correct?

Reply:
Good call, thoughts on proposal below?

Reply:
These look good!